script/tool/lib/src/version_check_command.dart

```diff
@@ -605,4 +613,46 @@ ${indentation}The first version listed in CHANGELOG.md is $fromChangeLog.
 
     return null;
   }
+
+  /// Returns true if [changeDescription] matches a Dependabot change for a
+  /// dependency roll that should bypass the normal version and CHANGELOG change
+  /// checks (for dependencies that are known not to have client impact).
+  bool _isAllowedDependabotChange(
+      RepositoryPackage package, String changeDescription) {
+    // Espresso exports some dependencies that are normally just internal test
+    // utils, so always require reviewers to check that.
+    if (package.directory.basename == 'espresso') {
+      return false;
+    }
+
+    // A string that is in all Dependabot PRs, but extreemly unlikely to be in
+    // any other PR, to identify Dependabot PRs.
+    const String dependabotMarker = 'Dependabot commands and options';
```

Comment:
I'm sure this should work in practice, but have you considered a more deterministic way to ensure a PR is a dependabot PR? It doesn't seem like you can configure the PR description, though, so this may be the best way for now.

Reply:
Agreed, this is not a great solution. I was hoping when I started implementing this that there would be a `CIRRUS_*` variable for the author, but unfortunately there isn't, so the only way I could see to get it would be much more complicated (taking the `CIRRUS_PR` and trying to extract the information from the GitHub API).

Given that, and that we can easily update the tool if Dependabot PR format changes, I figured this was probably a reasonable starting point, and then if it turns out to need a lot of maintenance in practice we can revisit. Longer term, I was thinking we may want to explore moving some checks like this one to infra's `cocoon` tool, which already has a lot of utilities for using the GitHub API to look at PR state; using PR description for doing overrides in the first place was always kind of a hack (and has proved less reliable than expected in other ways).

Since the failure case here is needing human intervention (rather than silently allowing incorrect changes), I'm more comfortable with it being more fragile than most of the repo tooling.